frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -120,15 +123,34 @@ export default ({
         },
         { type: 'header', name: 'Voting Systems' },
         // { type: 'item', id: 'change-disagreeing-number', name: 'Change disagreeing number', icon: 'vote-yea' },
-        { type: 'item', id: 'change-to-percentage-base', name: 'Change to percentage base', icon: 'vote-yea' },
+        {
+          type: 'item',
+          id: 'change-to-percentage-base',
+          name: 'Change to percentage base',
```

Comment:
While we're boy-scoutting, three change requests:

1. We forgot to wrap all of these in `L` functions, please update this line and the lines below for each `name` entry to wrap the string in an `L` function and re-run `strings`
2. Also, can you change this specific string from `'Change to percentage base'` to: `'Change voting threshold'` ?
3. Also wrap using `L` function the `name` for the `header` (e.g. `'Voting Systems'` => `L('Voting Systems')`  --- all user-facing strings must be localized